java/hello-thiagoyeds.java

```diff
@@ -1,5 +1,5 @@
 public class Hello {
     public static void main(String[] args) {
-        System.out.println("Hello Hacktoberfest 2022! My name is thiagoyeds!"); 
+        System.out.println("Hello Hacktoberfest 2021! My name is thiagoyeds!"); 
```

Comment:
Please, apply the suggestion

```suggestion
        System.out.println("Hello Hacktoberfest 2022! My name is thiagoyeds!"); 
```

Reply:
done